strongloop/loopback-connector-mqlight
# Refactor and enable tests

- Enable tests by removing test/describe.js
- Move tests into test/integration
- Modify initialize to trigger callback correctly after connecting
  successfully (was causing tests to hang)
- Refactor test initialization to use expect and config globally
- Refactor suites to setup/teardown tests properly
- Remove should in favour of chai/dirty-chai
- Remove unused devdeps blueblird and sinon

@qpresley PTAL

cc @strongloop/fa-db-connectors
